src/components/Editor.vue

```diff
@@ -262,12 +269,16 @@ export default {
 		displayed() {
 			return this.currentSession && this.active
 		},
-		renderMenus() {
+		renderRichEditorMenus() {
 			return this.contentLoaded
 				&& this.isRichEditor
 				&& !this.syncError
 				&& !this.readOnly
 		},
```

Comment:
Are we using this anywhere? Or are you just keeping it in case we need it later?
```suggestion
```

Reply:
I guess we still need this in earlier versions that still have the menu bubble. So good to keep it for now for the backports and clean it up separately.